kolibri/plugins/document_pdf_render/assets/src/views/index.vue

```diff
@@ -32,6 +32,7 @@
     data: () => ({
       supportsPDFs: PDFobject.supportsPDFs,
```

Comment:
This seems like a slightly odd use of the `data` attribute, unless we are planning to make them mutable? Would probably be better as computed properties.